features/device-config/persistence/api/pom.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>org.opennms.features.device-config.persistence</artifactId>
+        <groupId>org.opennms.features.device-config</groupId>
+        <version>30.0.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.opennms.features.device-config.persistence</groupId>
+    <artifactId>org.opennms.features.device-config.persistence.api</artifactId>
+    <packaging>bundle</packaging>
+    <name>OpenNMS :: Features :: Device Config :: Persistence :: API</name>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-RequiredExecutionEnvironment>JavaSE-1.8</Bundle-RequiredExecutionEnvironment>
```

Comment:
Should this really be 1.8?

Reply:
that's common across all pom for `maven-bundle-plugin`. Guess we should update all of them together

Reply:
sure, not a big deal, just a thing I noticed